tests/core/test_memory.py

```diff
@@ -163,16 +215,16 @@ def test_rnn_summary_shapes(mode):
         [b, t, i],  # rnn
         [b, t, h],  # linear
     ]
-    assert summary.out_sizes == [
-        [[b, t, h], [[1, b, h], [1, b, h]]],    # rnn
-        [b, t, o]                               # linear
-    ]
+    assert summary.out_sizes == [[[b, t, h], [[1, b, h], [1, b, h]]], [b, t, o]]  # rnn  # linear
```

Comment:
This change made the comments not indicate properly what is each part

Reply:
formatter :(

Reply:
You can avoid the formatting by adding the "Magic comma" https://github.com/psf/black/issues/1288 (`[b, t, o],`)

Reply:
@carmocca thanks, need to setup my new plugin :) will add it in the config.